### PR TITLE
[12.0][MIG] web_tree_many2one_clickable

### PR DESCRIPTION
**Pull request by Tardo**
This continue #1162 and adds #1041 features...

cc @Tecnativa

### CODE REVIEW

File: web_tree_many2one_clickable/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_tree_many2one_clickable
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This addon forces Odoo to use many2one widget on a many2one fields in
```
Comment (pedrobaeza) [inline web_tree_many2one_clickable/README.rst:28]: You need to adjust this text to the new behavior, and also a USAGE section with a screenshot of the button.

**Review by pedrobaeza — APPROVED** [review]
I love the new behavior!!

**Review by LoisRForgeFlow — APPROVED** [review]
Yes, much better than previous approach :+1:

**Review by chienandalu — APPROVED** [review]
:+1: Code and functional review. Thanks!

**Review by astirpe — APPROVED** [review]
Tested in Runbot :+1:

**Review by simahawk — APPROVED** [review]
bare code review, LG

**Pull request merged**